tests/tests_additional_packages.yml

```diff
@@ -26,4 +26,4 @@
     - name: Verify the packages were installed
       assert:
         that:
-          - "'{{ additional_package | trim }}' in ansible_facts.packages"
+          - "additional_package | trim in ansible_facts.packages"
```

Comment:
note that you do not need quotes here - this should work
```
          - additional_package | trim in ansible_facts.packages
```
same with the below

Reply:
Thanks! Updated.

Reply:
sounds like this does not work, at least not for the more complicated expressions as in `tests/tests_no_skip_defaults.yml`. But dropped the needless quotes from the rest of the ones.